### ISSUE TITLE
bug: (YT Music): Integrations patch failed on 7.01.52+

### ISSUE THREAD

**Issue by cyberboh** [ReVanced/revanced-patches#3172]
### Bug description

As title says, `GmsCore support` patch fail to be applied
Download APK here 
`https://www.apkmirror.com/apk/google-inc/youtube-music/youtube-music-7-01-52-release/youtube-music-7-01-52-android-apk-download/`
And patch with default patches
Using patches [v4.8.0-dev.18](https://github.com/ReVanced/revanced-patches/releases/tag/v4.8.0-dev.18) and Inte [v1.9.0-dev.10](https://github.com/ReVanced/revanced-integrations/releases/tag/v1.9.0-dev.10)

### Error logs

```shell
SEVERE: GmsCore support failed:
app.revanced.patcher.patch.PatchException: 'GmsCore support' depends on 'IntegrationsPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: Could not find hook target fingerprint.
        at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch$IntegrationsFingerprint.invoke(BaseIntegrationsPatch.kt:142)
        at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch.execute(BaseIntegrationsPatch.kt:41)
        at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch.execute(BaseIntegrationsPatch.kt:19)
        at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:173)
        at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:155)
        at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:205)
        at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
        at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
        at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61)
        at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230)
        at app.revanced.cli.command.PatchCommand$run$4$1$1.invokeSuspend(PatchCommand.kt:303)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:302)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)

        at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:160)
        at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:205)
        at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
        at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt)
        at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61)
        at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230)
        at app.revanced.cli.command.PatchCommand$run$4$1$1.invokeSuspend(PatchCommand.kt:303)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
        at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
        at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
        at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
        at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
        at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
        at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:302)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by kotenok2000** [comment]
6.51.52 fails too.

**Comment by kotenok2000** [comment]
play store installs this version for Redmi 10C

**Comment by soyDESKTOP68A2MRFT** [comment]
> > 6.51.52 fails too.
> 
> Really? Untested for backwards version, it could be dev patches issues

Actually i just tested on last stable (4.7.0)
And got this


`- Device Info
ReVanced Manager: 1.20.1
Model: SM-A125M
Android version: 12
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Root permissions: No

- Patch Info
App: com.google.android.apps.youtube.music v7.01.52 (Suggested: Any)
Patches version: v4.7.0
Patches added: Change package name
Patches removed: Hide 'Get Music Premium' label, Remove upgrade button
Default patch options changed: None

- Settings
Allow changing patch selection: true
Version compatibility check: false
Show universal patches: true
Remove unused library: true
Use pre-release: false
Patches source: revanced/revanced-patches
Integration source: revanced/revanced-integrations

- Logs
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing temporary files directory
Decoding resources
Executing patches
Applied 5 patches
GmsCore support failed: app.revanced.patcher.patch.PatchException: 'GmsCore support' depends on 'IntegrationsPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: Could not find hook target fingerprint.
	at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch$IntegrationsFingerprint.invoke(BaseIntegrationsPatch.kt:140)
	at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch.execute(BaseIntegrationsPatch.kt:39)
	at app.revanced.patches.shared.misc.integrations.BaseIntegrationsPatch.execute(BaseIntegrationsPatch.kt:17)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$34(Unknown Source:462)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$hsuHvwTCdC5Nf5dk5fKUQSHxuuY(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda5.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$34(Unknown Source:462)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$hsuHvwTCdC5Nf5dk5fKUQSHxuuY(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda5.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Compiling patched dex files
Canceling patching process
Press back to exit`

**Comment by soyDESKTOP68A2MRFT** [comment]
> > 6.51.52 fails too.
> 
> Really? Untested for backwards version, it could be dev patches issues

Also, i couldn't make the log have this codeish style, like in the comment that's why i edited it several times, because i was trying to do exactly that

**Comment by kotenok2000** [comment]
Maybe you should replace "Suggested: Any version" with something else?

**Comment by Szaamaan** [comment]
I think the last version that worked for me was `6.47.53`. All the versions after that had some kind of patching issue.

**Comment by Szaamaan** [comment]
No, I have not. I've not played around with anything but the official releases. But I'm testing other versions right now, just to make sure which one "breaks". I'm clearly wrong with my previous statement as I've just successfully patched `6.48.52`...

**Comment by Szaamaan** [comment]
I'll probably go with the "slightly older version" for now. 😉
